lib/attr_encrypted/adapters/active_record.rb

```diff
@@ -2,6 +2,9 @@
   module AttrEncrypted
     module Adapters
       module ActiveRecord
+        # Allows AR to infer the proper type for the column
+        EncryptedReflection = Struct.new(:klass)
```

Comment:
Are the reflections encrypted? I don't believe they are. If so, this name is misleading.